region/maridia/inner-pink.json

```diff
@@ -2562,6 +2665,72 @@
                     "canSuitlessMaridia",
                     "Grapple"
                   ]
+                },
+                {
+                  "name": "Cross Room Jump with IBJ",
+                  "notable": false,
+                  "requires": [
+                    "canIBJ",
```

Comment:
There's some variance on if the mochtroid stays close enough to do damage, possibly tied to how well you do the jump or turnaround.  2 hits if you do get grabbed and then your bombs kill it.